docs/get-started/cli.md

```diff
@@ -0,0 +1,105 @@
+---
+parent: Get started
+---
+
+# Build SDKs for CLI
+
+## Required tools
+A commandline tool is required. We recommend:
+- [Windows Terminal + version](https://apps.microsoft.com/store/detail/windows-terminal/9N0DX20HK701?hl=en-us&gl=us)
+- A commandline that supports Bash?
+- MacOS terminal
+- PowerShell version 7.3.2
+
+## Target project requirements
+Download the CLI SDK to get started for 
+Windows, Mac, or Linux here on the assets link of the latest release: [CLI SDK Download](https://github.com/microsoftgraph/msgraph-cli/releases)
+
+
+#
+## Adding dependencies
+
+### Getting access to the packages
+
+### Add Kiota and dependencies
+
+> **Note:** Find current version numbers for Kiota packages at [Nexus Repository Manager](https://oss.sonatype.org/).
+
+
+## Generating the SDK
+
+Kiota generates SDKs from OpenAPI documents. Create a file named **getme.yml** and add the contents of the [Sample OpenAPI description](reference-openapi.md).
+
+You can then use the Kiota command line tool to generate the SDK classes.
+
+```bash
+#TBD
+```
+
+## Registering an application in Azure AD
+
+## Creating an application registration
+
+> **Note:** this step is required if your client will be calling APIs that are protected by the Microsoft Identity Platform like Microsoft Graph.
+
+Follow the instructions in [Register an application for Microsoft identity platform authentication](register-app.md) to get an application ID (also know as a client ID).
+
+## Creating the client application
+
+Replace `YOUR_CLIENT_ID` with the client ID from your app registration.
+
+``` bash
+mgc login --client-id `YOUR_CLIENT_ID` --tenant-id `YOUR_TENANT_ID` --scopes User.ReadWrite --scopes Mail.ReadWrite

```

Comment:
mgc is going to depend on the project configuration, we should call out that configuration

Reply:
Closed by this PR https://github.com/microsoft/kiota/pull/2298